src/main/java/discord4j/discordjson/json/InviteData.java

```diff
@@ -16,23 +16,41 @@
     }
 
     String code();
+
     Possible<PartialGuildData> guild();
+
     ChannelData channel();
+
+    Possible<UserData> inviter();
+
+    @JsonProperty("target_type")
+    Possible<Integer> targetType();
+
     @JsonProperty("target_user")
     Possible<UserData> targetUser();
-    @JsonProperty("target_user_type")
-    Possible<Integer> targetUserType();
+
+    /* TODO
+     * @JsonProperty("target_application")
+     * Possible<?> targetApplication();
+     */
```

Comment:
Is this because we don't have any application stuff? Do we need to do anything about that (can be done separately if so)?

Reply:
The `target_application` object returned is a `partial application` object
There is no example of what it could be or what are the fields and I didn't try myself to see